kaggle/RANZCR/4th_place_solution/README.md

```diff
@@ -26,36 +28,53 @@ Please download and put it into the dataset's path
 
 A single NVIDIA Tesla V100 32G
 
+# Inference
+
+To do inference locally, the command could be:
+
+```
+python train.py -c cfg_seg_40_1024d_full -i True -p your_actual_local_weight_folder_path
+```
+
+In Kaggle, since the kernel for submission does not support the internet access, you may need to copy the code of `MONAI/monai/` (if using MONAI docker, the path is `/opt/monai/monai/`) into this directory and upload the whole folder as a Kaggle dataset to use. Please take the `run_infer` function in `train.py` for reference and write the script.
+
+# Places uncovered
```

Comment:
I misunderstood the term here. I think it should be `Strategies used in the solutions`